### ISSUE TITLE
Transformer Instance: Cannot use object of type MyTransformerName as array

### ISSUE THREAD

**Issue by handlebarcreative** [pixelandtonic/ElementAPI#1]
Less of an issue, more of a documentation update.

The docs make mention of using a class instance for a transformer.

```
// Or a Transformer class instance
'transformer' => new MyTransformerClassName(),
```

Having no experience with Fractal, I found this a bit confusing, and was always getting an error returned reading: "Fatal error: Cannot use object of type Craft\MyTransformerClassName as array in /Users/george/myCraftApp/craft/app/framework/YiiBase.php on line 183".  
I figured it out in the end - but not sure if my solution was the best way tackle it.  Would love your feedback, and a possible docs update.

In my custom class transformer, I had to extend `League\Fractal\TransformerAbstract` as follows:

In `/craft/config/MyTransformerClassName.php`:

```

    require craft()->path->getPluginsPath().'elementapi/vendor/autoload.php';
    use League\Fractal\TransformerAbstract;
    class MyTransformerClassName extends TransformerAbstract{
        public function transform(EntryModel $entry) {
            return [
                'slug' => $entry->slug,
                //... more custom props
            ];
        }
    }
```

**Comment by mobyjames** [comment]
Agree. I didn't figure it out until I read this.

**Comment by takobell** [comment]
Fixed in 5e18cb748fc0ecdde2497fa7b035e317ff080636
